src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -304,8 +304,11 @@ public string SearchText
             get { return _SearchText; }
             set
             {
-                _SearchText = value;
-                RaisePropertyChanged("SearchText");
+                if(_SearchText!= value)
```

Comment:
Should there be a space here? "if(_SearchText != value)" ?

Reply:
It would work, but it would be cleaner :)) I will add that in, thanks for the hawk eyes @Amoursol !